cns/restserver/util.go

```diff
@@ -449,7 +449,7 @@ func (service *HTTPRestService) getNetworkContainerResponse(
 func (service *HTTPRestService) restoreNetworkState() error {
 	logger.Printf("[Azure CNS] Enter Restoring Network State")
 
-	if service.store == nil {
+	if service.store == nil || !service.store.Exists() {
```

Comment:
can you add this as separate if and log it so that we know if store is nil or state file doesn't exist